assets/src/components/button/style.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Button styles,
+ */
+
+
+.newspack-button {
+	.components-button {
+	    padding: 0 20px 1px;
+		text-align: center;
+
+		font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		line-height: 18px;
+		font-weight: 500;
+		color: $muriel-gray-700;
+		text-shadow: none;
+
+		&.is-button {
+			background-color: $muriel-white;
+			border: none;
+			border-radius: 3px;
+			cursor: pointer;
+			height: 40px;
+			max-width: 360px;
+			max-height: 40px;
+			margin-top: 13px;
+			margin-bottom: 9px;
+
+			&:hover {
+				border-color: $muriel-gray-500;
+			}
+
+			&:active:enabled {
+				border-width: 2px 1px 1px;
+			}
+		}
+
+		&.is-primary {
+			background-color: $muriel-hot-pink-500;
+			border-color: #992053;
+			box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+			color: $muriel-white;
+
+			&:hover,
+			&:focus:enabled {
+				background-color: $muriel-hot-pink-400;
+				color: $muriel-gray-0;
+			}
+
+			&:active:enabled {
+				background-color: $muriel-hot-pink-700;
+				color: $muriel-white;
+			}
+		}
+
+		&.is-tertiary {
```

Comment:
Seems to me there are two ways to go with these components: 1) support all the variants that the Gutenberg component offers, making this a truly robust and reusable implementation, or 2) start out by supporting only the use case we need and add the other variants later. We've been taking the pragmatic 2nd approach in order to get moving quickly, and as we develop confidence around the components we'll go back fill in the gaps. Robust support for `Button` would mean, at minimum, supporting `isPrimary | isLarge | isSmall | isTertiary | isToggled | isBusy | isDefault | isLink | isDestructive`, although I believe the only one the Newspack mockups call for is `isPrimary`. In the spirit of this, let's limit our styles to the required variant(s) to avoid confusion later. All of which is a long-winded way of saying, let's remove the `is-tertiary` block.

Reply:
Agreed, we only have 2 button styles in our designs from what I can tell: primary and secondary.